settings.coffee

```diff
@@ -438,6 +438,7 @@ if process.env["SHARELATEX_SAML_ENTRYPOINT"]
 			callbackUrl: process.env["SHARELATEX_SAML_CALLBACK_URL"]
 			issuer: process.env["SHARELATEX_SAML_ISSUER"]
 			decryptionPvk: process.env["SHARELATEX_SAML_DECRYPTION_PVK"]
+      decryptionCert: process.env["SHARELATEX_SAML_DECRYPTION_CERT"]
```

Comment:
```suggestion
			decryptionCert: process.env["SHARELATEX_SAML_DECRYPTION_CERT"]
```

Reply:
Thanks! Would you mind signing the [Contributor Agreement](https://docs.google.com/forms/d/19JsUGMlhM5LJ0mA4Vj_guMiSoX1pNg_-KvvZT5-H0LQ) before proceeding to include your suggestion?

https://docs.google.com/forms/d/19JsUGMlhM5LJ0mA4Vj_guMiSoX1pNg_-KvvZT5-H0LQ

Reply:
Sorry, I don't sign CLAs. Didn't realize the review was gonna make a problem for that. If it helps, I'm licensing it under CC0.
If it's that much of a problem, let me know an I'll sign it for this time.